src/py-opentimelineio/opentimelineio-bindings/otio_serializableObjects.cpp

```diff
@@ -203,20 +203,8 @@ static void define_bases2(py::module m) {
     MarkerVectorProxy::define_py_class(m, "MarkerVector");
     EffectVectorProxy::define_py_class(m, "EffectVector");
 
-    py::class_<Composable, SOWithMetadata,
```

Comment:
Moved into the `define_items_and_compositions` function since the `parent` method returns a Composition which is defined in `define_items_and_compositions`.